stylecop.ruleset

```diff
@@ -1,102 +1,97 @@
-    <Rule Id="CA1065" Action="Warning"/>
-    <Rule Id="CA1301" Action="Warning"/>
-    <Rule Id="CA1400" Action="Warning"/>
-    <Rule Id="CA1401" Action="Warning"/>
-    <Rule Id="CA1403" Action="Warning"/>
-    <Rule Id="CA1404" Action="Warning"/>
-    <Rule Id="CA1405" Action="Warning"/>
-    <Rule Id="CA1410" Action="Warning"/>
-    <Rule Id="CA1415" Action="Warning"/>
-    <Rule Id="CA1724" Action="None"/>
-    <Rule Id="CA1801" Action="Warning"/>
-    <Rule Id="CA1821" Action="Warning"/>
-    <Rule Id="CA1822" Action="None"/>
-    <Rule Id="CA1900" Action="Warning"/>
-    <Rule Id="CA1901" Action="Warning"/>
-    <Rule Id="CA2002" Action="Warning"/>
-    <Rule Id="CA2100" Action="Warning"/>
-    <Rule Id="CA2101" Action="Warning"/>
-    <Rule Id="CA2108" Action="Warning"/>
-    <Rule Id="CA2111" Action="Warning"/>
-    <Rule Id="CA2112" Action="Warning"/>
-    <Rule Id="CA2114" Action="Warning"/>
-    <Rule Id="CA2116" Action="Warning"/>
-    <Rule Id="CA2117" Action="Warning"/>
-    <Rule Id="CA2122" Action="Warning"/>
-    <Rule Id="CA2123" Action="Warning"/>
-    <Rule Id="CA2124" Action="Warning"/>
-    <Rule Id="CA2126" Action="Warning"/>
-    <Rule Id="CA2131" Action="Warning"/>
-    <Rule Id="CA2132" Action="Warning"/>
-    <Rule Id="CA2133" Action="Warning"/>
-    <Rule Id="CA2134" Action="Warning"/>
-    <Rule Id="CA2137" Action="Warning"/>
-    <Rule Id="CA2138" Action="Warning"/>
-    <Rule Id="CA2140" Action="Warning"/>
-    <Rule Id="CA2141" Action="Warning"/>
-    <Rule Id="CA2146" Action="Warning"/>
-    <Rule Id="CA2147" Action="Warning"/>
-    <Rule Id="CA2149" Action="Warning"/>
-    <Rule Id="CA2200" Action="Warning"/>
-    <Rule Id="CA2202" Action="Warning"/>
-    <Rule Id="CA2207" Action="Warning"/>
-    <Rule Id="CA2212" Action="Warning"/>
-    <Rule Id="CA2213" Action="Warning"/>
-    <Rule Id="CA2214" Action="Warning"/>
-    <Rule Id="CA2216" Action="Warning"/>
-    <Rule Id="CA2220" Action="Warning"/>
-    <Rule Id="CA2229" Action="Warning"/>
-    <Rule Id="CA2231" Action="Warning"/>
-    <Rule Id="CA2232" Action="Warning"/>
-    <Rule Id="CA2235" Action="Warning"/>
-    <Rule Id="CA2236" Action="Warning"/>
-    <Rule Id="CA2237" Action="Warning"/>
-    <Rule Id="CA2238" Action="Warning"/>
-    <Rule Id="CA2240" Action="Warning"/>
-    <Rule Id="CA2241" Action="Warning"/>
-    <Rule Id="CA2242" Action="Warning"/>
-    <!-- UNDONE: produces false positives in Microsoft.CodeAnalysis.FxCopAnalyzers.2.3.0-beta1 -->
-    <Rule Id="CA1812" Action="None"/>
-    <Rule Id="CA1823" Action="None"/>
-  </Rules>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Microsoft Managed Recommended Rules" Description="These rules focus on the most critical problems in your code, including potential security holes, application crashes, and other important logic and design errors. It is recommended to include this rule set in any custom rule set you create for your projects." ToolsVersion="15.0">
```

Comment:
Why are you still changing entire configuration instead of adding your feature? Please check what you commit.